app/jobs/import_block_job.rb

```diff
@@ -1,4 +1,4 @@
-class ProcessBlockJob < ApplicationJob
+class ImportBlockJob < ApplicationJob
```

Comment:
When to invoke this job?

Reply:
This is a placeholder for future refactoring